### PR TITLE
todo task for gci to elaborate fields

### PR DESCRIPTION
**Pull request by samsruti**
Todo Task for GCI task : https://www.google-melange.com/gci/task/view/google/gci2014/5038048144261120
Also added a field , "Identity proof", suppose if some found any id / driving license/or any visitor card , he can also give details and that will help to find out the dead person's family

### CODE REVIEW

File: modules/s3db/dvi.py [hunk 1]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
-                     Field("clothing", "text"),  # @todo: elaborate
-                     Field("jewellery", "text"), # @todo: elaborate
-                     Field("footwear", "text"),  # @todo: elaborate
-                     Field("watch", "text"),     # @todo: elaborate
-                     Field("other", "text"),
+                     Field("identity_prrof", "text", length=64,
+                           label = T("Identity Proof"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Identity Proof"),
+                                                           T("If any identity prrof like ID card , driving license or anything else , Write the Details"))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: prrof -> proof. Please correct.

Comment (ramdesh) [inline modules/s3db/dvi.py]: And change to "If any proof of identity such as Identity Card or Driving License is found, add details here"

File: modules/s3db/dvi.py [hunk 2]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                          ),                      
+                     Field("clothing", "text", length=120,
+                           label = T("Clothing"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
+                                                           T("Decribe the clothing Details of the dead body"))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: De-capitalize Details

File: modules/s3db/dvi.py [hunk 3]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           label = T("Jewellery"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Jewellery"),
+                                                           T("If any jewellery found , give the details"))),
+                           ),
+                     Field("footwear", "text",length=60,
+                           label = T("Footwear"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Footwear"),
+                                                           T("Decribe about the footwear of the dead body like size of the footwear , type or any other"))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: "Details of footwear the person is wearing"

File: modules/s3db/dvi.py [hunk 4]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           ),
+                     Field("watch", "text",length=60,
+                           label = T("Watch"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Watch"),
+                                                           T("If any watch found , give the details of watch like colour , brand"))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: If a watch was found on the person, give the details of watch like colour, brand

File: modules/s3db/dvi.py (hunk 1 above)
Comment (samsruti) [inline modules/s3db/dvi.py]: Sorry for the spellings and sentences.

File: modules/s3db/dvi.py [hunk 5]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           ), 
+                     Field("other", "text",length=300,
+                           label = T("Other"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
+                                                           T("Give the details, if any other material found from the dead body ."))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: Give details of any other material found on the person.

File: modules/s3db/dvi.py [hunk 6]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           label = T("Clothing"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
+                                                           T("Decribe the clothing details of the dead body"))),
+                          ), 
+                     Field("jewellery", "text",length=120,
+                           label = T("Jewellery"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Jewellery"),
+                                                           T("If any jewellery found , give the details"))),
```
Comment (ramdesh) [inline modules/s3db/dvi.py]: Give details of any jewellery found on the person.

File: modules/s3db/dvi.py [hunk 7]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                                         _title="%s|%s" % (T("Identity Proof"),
+                                                           T("If any proof of identity such as Identity Card or Driving License is found, add details here ."))),
+                          ),                      
+                     Field("clothing", "text", length=120,
+                           label = T("Clothing"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
+                                                           T("Decribe the clothing details of the dead body ."))),
+                          ), 
+                     Field("jewellery", "text",length=120,
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: Space after ','

File: modules/s3db/dvi.py [hunk 8]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           label = T("Jewellery"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Jewellery"),
+                                                           T("Give details of any jewellery found on the person ."))),
+                           ),
+                     Field("footwear", "text",length=60,
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: Space after ','

File: modules/s3db/dvi.py [hunk 9]
```diff
@@ -344,11 +344,42 @@ def model(self):
...
+                           label = T("Footwear"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Footwear"),
+                                                           T("Give Details of footwear the person is wearing ."))),
+                           ),
+                     Field("watch", "text",length=60,
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: Everywhere else too.

File: modules/s3db/dvi.py [hunk 10]
```diff
@@ -344,11 +344,48 @@ def model(self):
...
+                           label = T("Watch"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Watch"),
+                                                           T("If a watch was found on the person, give the details of watch like colour, brand ."))),
+                           ), 
+                     Field("other", "text",
+                           length=300,
+                           label = T("Other"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: Clothing ?

File: modules/s3db/dvi.py [hunk 11]
```diff
@@ -344,11 +344,48 @@ def model(self):
...
+                           ),
+                     Field("watch", "text",
+                           length=60,
+                           label = T("Watch"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Watch"),
+                                                           T("If a watch was found on the person, give the details of watch like colour, brand ."))),
+                           ), 
+                     Field("other", "text",
+                           length=300,
```
Comment (hitesh96db) [inline modules/s3db/dvi.py:383]: <pre>
length = 300
</pre>

Same everywhere else

File: modules/s3db/dvi.py [hunk 12]
```diff
@@ -344,11 +344,48 @@ def model(self):
...
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Clothing"),
+                                                           T("Decribe the clothing details of the dead body ."))),
+                          ), 
+                     Field("jewellery", "text",
+                           length=120,
+                           label = T("Jewellery"),
+                           comment = DIV(_class="tooltip",
+                                         _title="%s|%s" % (T("Jewellery"),
+                                                           T("Give details of any jewellery found on the person ."))),
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: No need of space between person and "."
Same everywhere else

File: modules/s3db/dvi.py [hunk 13]
```diff
@@ -344,11 +344,48 @@ def model(self):
         tablename = "dvi_effects"
         define_table(tablename,
                      super_link("pe_id", "pr_pentity"),
-                     Field("clothing", "text"),  # @todo: elaborate
-                     Field("jewellery", "text"), # @todo: elaborate
-                     Field("footwear", "text"),  # @todo: elaborate
-                     Field("watch", "text"),     # @todo: elaborate
-                     Field("other", "text"),
+                     Field("identity_proof", "text", 
+                           length=64,
```
Comment (hitesh96db) [inline modules/s3db/dvi.py]: Is there any reason why the length of this is '64' and the others 120 ?. Keep all of them 120 ? or 300 ?(since it's text)

**Pull request merged**